applications/CompressiblePotentialFlowApplication/tests/naca0012_small_compressible_test/naca0012_test_probe1_ref.dat

```diff
@@ -0,0 +1,3 @@
+# Results for "element" with Id # 21 at position: x: 1; y: 0.5; z: 0
+# time VELOCITY_POTENTIAL
```

Comment:
Same comment. 1 extra file to store 3 lines.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668